File: LinkedList/LinkedList-Introduction/Intro.java
class Intro{
static class Node {
    int data;    
    Node next;      
    Node(int data1) {
        this.data = data1;
        this.next = null;   
    }
}
    public static void main(String[] args) {
        int[] arr = {2, 5, 8, 7};
        // Creating a new Node with the value from the array
        Node y = new Node(arr[0]);
        // Printing the data stored in the Node
        System.out.println(y.data);
    }

}

File: LinkedList/01_Introduction/intro.cpp
#include<iostream>
#include<vector>
using namespace std;
class Node{
    public: 
    int data; 
    Node* next; 
    public:
    // constructor
    Node (int data1, Node* next1){
        data=data1;  
        next=next1; 
    }
    Node (int data1){
        data=data1; 
        next=nullptr; 

    }
};
int main(){
    vector<int> arr={2,5,8,7};
    Node* y= new Node(arr[0]);
    cout<<y<<endl; // returns the memory value
    cout<<y->data<<endl; // returns the data stored at that memory point
    return 0;
}